.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,29 +3,25 @@
 
 ### Pull Request Checklist
 
-Please check all boxes, including N/A items:
-
-#### Testing
-
-- [ ] Unit test and/or integration test added
-- [ ] __Toolkit change?:__ [integration tests](https://github.com/awslabs/aws-cdk/blob/master/packages/aws-cdk/integ-tests/test.sh) manually executed (paste output to the PR description)
-- [ ] __Init template change?:__ coordinated update of integration tests (currently maintained in a private repo).
-
-#### Documentation
-
-- [ ] __README__: README and/or documentation topic updated
-- [ ] __jsdocs__: All public APIs documented
-
-### Title and description
-
-- [ ] __Change type__: Title is prefixed with change type:
-  * `fix(module): <title>` bug fix (_patch_)
-  * `feat(module): <title>` feature/capability (_minor_)
-  * `chore(module): <title> ` won't appear in changelog
-  * `build(module): <title>` won't appear in changelog
-- [ ] __Title format__: Title uses lower case and doesn't end with a period
-- [ ] __Breaking change?__: Last paragraph of description is: `BREAKING CHANGE: <describe exactly what changed and how to achieve similar behavior + link to documentation/gist/issue if more details are required>`
-- [ ] __References__: Indicate issues fixed via: `Fixes #xxx` or `Closes #xxx`
+- [ ] Testing
+  - Unit test added
```

Comment:
I like to use different bullet styles for different levels, so it is easier to read when you're viewing it as markdown source (for example - when you're preparing your PR message). It's somewhat cosmetic though.

Reply:
Done. Good point